### PR TITLE
feat(charts): add an option to configure revisionHistoryLimit to subcharts

### PR DESCRIPTION
**Pull request by bskim45**
This PR adds the ability for end-users to configure `revisitonHistoryLimit` for the deployments of subcharts.

In addition, bumps chart version to 0.2.96

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by szalai1 — APPROVED** [review]
Thanks for adding this. LGTM

**Comment by bskim45** [comment]
Rebumped chart version to 0.2.94

File: charts/datahub/Chart.yaml [hunk 1]
```diff
@@ -4,25 +4,25 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.93
+version: 0.2.94
```
Comment (bskim45) [inline charts/datahub/Chart.yaml:7]:
```suggestion
version: 0.2.95
```

**Review by anshbansal — CHANGES_REQUESTED** [review]
As you have made changes in charts for both `datahub-frontend` as well as `datahub-gms` the versions for them also need to be increased by a minor version

**Comment by bskim45** [comment]
> As you have made changes in charts for both `datahub-frontend` as well as `datahub-gms` the versions for them also need to be increased by a minor version

done!

**Review by anshbansal — APPROVED** [review]
LGTM

**Pull request merged**